instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py

```diff
@@ -179,6 +191,21 @@ def instrumented_urlopen(wrapped, instance, args, kwargs):
     )
 
 
+def _call_request_hook(
+    request_hook: typing.Union[_RequestHookT, _ExtendedRequestHookT],
+    span: Span,
+    connection_pool: urllib3.connectionpool.HTTPConnectionPool,
+    headers: typing.Dict,
+    body: str,
+):
+    try:
+        # First assume request_hook is a function of type _ExtendedRequestHookT
+        request_hook(span, connection_pool, headers, body)
```

Comment:
Any reason to not always pass headers and body to hooks? I think its OK to break this since we aren't 1.0 yet.

Reply:
the only reason is backward compatibility, but if it's OK to break this, I'll change it to always pass these args